src/debug/ee/debugger.cpp

```diff
@@ -2113,18 +2113,9 @@ HRESULT Debugger::Startup(void)
             ShutdownTransport();
             ThrowHR(hr);
         }
-
     #ifdef FEATURE_PAL
         PAL_SetShutdownCallback(AbortTransport);
     #endif // FEATURE_PAL
-
-         bool waitForAttach = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_DbgWaitForDebuggerAttach) != 0;
```

Comment:
This change is a lot more related than the unixinterface.cpp/ExecuteAssembly change.  It needs to be removed and related to the attach scenario.

From: noahfalk [mailto:notifications@github.com]
Sent: Sunday, May 22, 2016 5:18 PM
To: dotnet/coreclr <coreclr@noreply.github.com>
Cc: Mike McLaughlin <mikem@microsoft.com>; Assign <assign@noreply.github.com>
Subject: Re: [dotnet/coreclr] Change the dbgshim launch handshake back (#5151)


In src/debug/ee/debugger.cpp<https://github.com/dotnet/coreclr/pull/5151#discussion_r64159356>:

>      #ifdef FEATURE_PAL

>          PAL_SetShutdownCallback(AbortTransport);

>      #endif // FEATURE_PAL

> -

> -         bool waitForAttach = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_DbgWaitForDebuggerAttach) != 0;

Does this edit relate to the bug fix? I'm not opposed to cleanup, but I don't know where the bar is now. If this is an unrelated code change you might be asked to split it out into a seperate commit that won't get integrated to RTM.

—
You are receiving this because you were assigned.
Reply to this email directly or view it on GitHub<https://github.com/dotnet/coreclr/pull/5151/files/41ca2d79992390fa2847629f34c30479644a9b7d#r64159356>